.github/workflows/_python-tests.yml

```diff
@@ -0,0 +1,111 @@
+name: Chroma Python Base Tests
+
+on:
+  workflow_call:
+    inputs:
+      python_versions:
+        description: 'Python versions to test (as json array)'
+        required: false
+        default: '["3.8"]'
+        type: string
+
+
+jobs:
+  test:
+    timeout-minutes: 90
+    strategy:
+      matrix:
+        python: ${{fromJson(inputs.python_versions)}}
+        platform: [ubuntu-latest, windows-latest]
+        test-globs: ["--ignore-glob 'chromadb/test/property/*' --ignore-glob 'chromadb/test/stress/*' --ignore-glob 'chromadb/test/distributed/*' --ignore='chromadb/test/auth/test_simple_rbac_authz.py'",
+                   "chromadb/test/auth/test_simple_rbac_authz.py",
+                   "chromadb/test/property/test_add.py",
+                   "chromadb/test/property/test_collections.py",
+                   "chromadb/test/property/test_cross_version_persist.py",
+                   "chromadb/test/property/test_embeddings.py",
+                   "chromadb/test/property/test_filtering.py",
+                   "chromadb/test/property/test_persist.py"]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/python-test
+        with:
+          python-version: ${{ matrix.python }}
+          test-globs: ${{ matrix.test-globs }}
+
+  test-single-node-integration:
+    strategy:
+      matrix:
+        python: ${{fromJson(inputs.python_versions)}}
+        platform: [ubuntu-latest] # todo: should run on Windows, currently failing because Dockerfile doesn't build
```

Comment:
this was previously silently failing, a fix seems like it could be a good chunk of work and is out of scope of this PR

learning: you **must** specify `shell: bash` in the Action config for a shell command when using `run` if you want `set -e` to properly take effect on Windows